src/treemap/index.js

```diff
@@ -114,47 +125,57 @@ class Treemap extends React.Component {
    * @private
    */
   _getNodesToRender() {
-    const {data, height, width, mode, padding} = this.props;
-
+    const {data, height, width, mode, padding, useCirclePacking} = this.props;
+    if (data && useCirclePacking) {
```

Comment:
ill add tests accordingly once the test pr lands, i'd rather not run into tons of conflicts